cmd/kpod/build.go

```diff
@@ -0,0 +1,139 @@
+package main
+
+import (
+	"os"
+	"os/exec"
+
+	"github.com/pkg/errors"
+	"github.com/urfave/cli"
+)
+
+var (
+	buildFlags = []cli.Flag{
+		cli.StringSliceFlag{
+			Name:  "build-arg",
+			Usage: "`argument=value` to supply to the builder",
+		},
+		cli.StringSliceFlag{
+			Name:  "file, f",
+			Usage: "`pathname or URL` of a Dockerfile",
+		},
+		cli.StringFlag{
+			Name:  "format",
+			Usage: "`format` of the built image's manifest and metadata",
+		},
+		cli.BoolFlag{
+			Name:  "pull-always",
+			Usage: "pull the image, even if a version is present",
+		},
+		cli.BoolFlag{
+			Name:  "quiet, q",
+			Usage: "refrain from announcing build instructions and image read/write progress",
+		},
+		cli.StringFlag{
+			Name:  "runtime",
+			Usage: "`path` to an alternate runtime",
+		},
+		cli.StringSliceFlag{
+			Name:  "runtime-flag",
+			Usage: "add global flags for the container runtime",
+		},
+		cli.StringFlag{
+			Name:  "signature-policy",
+			Usage: "`pathname` of signature policy file (not usually used)",
+		},
+		cli.StringSliceFlag{
+			Name:  "tag, t",
+			Usage: "`tag` to apply to the built image",
+		},
+		cli.BoolFlag{
+			Name:  "tls-verify",
+			Usage: "Require HTTPS and verify certificates when accessing the registry",
+		},
+	}
+	buildDescription = "This command launches the Buildah command to build an OCI Image. Buildah must be installed for this command to work."
```

Comment:
How does this look when you do a buildah build --help?

Reply:
No help topic.  Should we add 'build' as another buildah command? 

```
# buildah build --help
No help topic for 'build'
[root@localhost ~]# buildah bud --help
NAME:
   buildah build-using-dockerfile - Build an image using instructions in a Dockerfile

USAGE:
   buildah build-using-dockerfile [command options] CONTEXT-DIRECTORY | URL
{TAS: removed more output}
```

Reply:
Ooops
I meant podman build --help.

Reply:
I haven't yet merged into podman, but here's what comes up with kpod build --help.

```
# kpod build --help
NAME:
   kpod build - Build an image using instructions in a Dockerfile

USAGE:
   kpod build [command options] CONTEXT-DIRECTORY | URL

DESCRIPTION:
   This command launches the Buildah command to build an OCI Image. Buildah must be installed for this command to work.

OPTIONS:
   --build-arg argument=value                  argument=value to supply to the builder
   --file pathname or URL, -f pathname or URL  pathname or URL of a Dockerfile
   --format format                             format of the built image's manifest and metadata
   --pull-always                               pull the image, even if a version is present
   --quiet, -q                                 refrain from announcing build instructions and image read/write progress
   --runtime path                              path to an alternate runtime
   --runtime-flag value                        add global flags for the container runtime
   --signature-policy pathname                 pathname of signature policy file (not usually used)
   --tag tag, -t tag                           tag to apply to the built image
   --tls-verify                                Require HTTPS and verify certificates when accessing the registry
 ```